Tic Tac Toe.py

```diff
@@ -0,0 +1,175 @@
+    elif player2 == 1:
+        print('Player1 `s turn')
+        return (1, 0)
+
+def main():
+    while True:
+        next_game = True
+        while next_game:
+            next_game = False
+            print_template()
+            player1, player2 = first_player()
+            mark_player1, mark_player2 = choose_sign(player1, player2)
+            score = {'Player1': 0, 'Player2': 0}
+            count = 0
+            while count < 3:
+                count += 1
+                while True:
+                    if player1 == 1:
+                        try:
+                            inpt = input('Player1 please insert position: ')
+                            if dic[inpt] == ' ':
+                                dic[inpt] = mark_player1
+                                print_progress(dic)
+                                if win(dic, mark_player1):
+                                    print('Player1 is winner')
+                                    score['Player1'] += 1
+                                    break
+                            else:
+                                print('Possition is occupied ')
+                                continue
+                        except :
+                            print('You have to insert integer 1-9 !')
+                            continue
+                        player1, player2 = 0, 1
+
+                    elif player2 == 1:
+                        try:
+                            inpt = input('Player2 please insert position: ')
+                            if dic[inpt] == ' ':
+                                dic[inpt] = mark_player2
+                                print_progress(dic)
+                                if win(dic, mark_player2):
+                                    print('Player2 is winner')
+                                    score['Player2'] += 1
+                                    break
+                            else:
+                                print('Possition is occupied ')
+                                continue
+                        except :
+                            print('You have to insert integer 1-9 !')
+                            continue
+                        player1, player2 = 1, 0
+                print('Score is: Player1:{}   Player2:{} \n'.format(score['Player1'], score['Player2']))
+
+                if count < 3:
+                    exit_request = input('Do you want to play next round ? y=yes, anything for no  ')
+                    player1, player2 = exit_requests(player1, player2)
+                    if exit_request == 'y':
+                        continue
+                    else:
+                        break
+
+        try:
+            game_input = input('Do you want to play next game ? y=yes, anything for no  ')
+            if game_input == 'y':
+                for i in dic:
+                    dic[i] = ' '
+                next_game = True
+            else:
+                return None
```

Comment:
You don't have to explicitly return `None` if you don't care about it. Just use `return` without return value.